src/lib/validations/sql.js

```diff
@@ -118,6 +118,15 @@ const checks: Checks = {
 		excerpt: '\'CREATE DATABASE\' statement should not  be present (case-insensitive)',
 		recommendation: 'Remove these lines',
 	},
+	binaryLogging: {
+		matcher: /SET @@SESSION.sql_log_bin/i,
+		matchHandler: lineNumber => lineNumber,
+		outputFormatter: errorCheckFormatter,
+		results: [],
+		message: 'SET @@SESSION.sql_log_bin statement',
+		excerpt: '\'SET @@SESSION.sql_log_bin=0\' statement should not be present (case-insensitive)',
```

Comment:
Super minor, but we probably don't need the `=0` here.

Reply:
Oh yeah good catch!

Reply:
Fixed!